pandas/core/series.py

```diff
@@ -2861,6 +2864,8 @@ def create_from_value(value, index, dtype):
 
         if is_datetimetz(dtype):
```

Comment:
you may need to make this a single routine something like in ``pandas.types.dtypes```

```
def _create_array(arr, dtype):
     if is_extension_dtype(dtype):
             # _type_class is the Index type, but not sure how this should be attached to dtype.
             return dtype._type_class(arr)
     return np.asarray(arr, dtype=dtype)

```